scripts/release/mirror.js

```diff
@@ -29,9 +29,13 @@ export const getMatchingBrowserVersion = (targetBrowser, sourceVersion) => {
   for (const r of releaseKeys) {
     const release = browserData.releases[r];
     if (
-      ['opera', 'opera_android', 'samsunginternet_android'].includes(
-        targetBrowser,
-      ) &&
+      [
+        'edge',
```

Comment:
This also provides a fix for Edge's mirroring if attempting to mirror a pre-Blink version of Chrome to Edge.  I didn't feel it was worth placing in a separate PR though.